app/components/MobileMockup.tsx

```diff
@@ -23,12 +23,18 @@ const MobileMockup = ({ imageSrc, isHomepage = false }: MockupProps) => {
       <Image
         src={iphoneTemplate}
         alt="iphone template image"
-        className="z-20"
+        className="z-20 h-auto w-auto"
       />
       <div
         className={`absolute left-1 top-1 z-10 flex h-[6.5rem] w-[3.5rem] overflow-hidden rounded-md ${childDivSytles}`}
       >
-        <Image src={imageSrc} alt="mobile image of project" objectFit="cover" />
+        <Image
+          src={imageSrc}
+          alt="mobile image of project"
+          objectFit="cover"
+          width={300}
+          height={300}
```

Comment:
I don't believe the width and height are necessary here. Per the guidelines, `object-fit: "cover"` will cause the image to fill the entire container and be cropped to preserve the aspect ratio.

Reply:
@GlenMac90, I believe I have found the solution. Take a look at the link below that explains it all in good detail. I will also include the NPM module that is referenced. This will be a great way to practice making a separate request to address this one specific issue. :)

- take your time to do a little research as well. It's important that we implement this correctly. It's task such as these that make you a expert developer. 

https://dev.to/fibonacid/practical-image-rendering-guide-for-sanity-and-nextjs-3gc7
https://www.sanity.io/docs/image-urls
https://www.sanity.io/docs/image-url
The npm module referenced: npm i @sanity/image-url